content/home/intro.md

```diff
@@ -4,4 +4,10 @@
 - SLO-driven multi-stage delivery 
 - Operations & remediation
 
-declarative, extensible and based on GitOps
\ No newline at end of file
+declarative, extensible and based on GitOps
+
+### Cloud Native?
```

Comment:
i think we also need to adapt our catch phrase then

![image](https://user-images.githubusercontent.com/9987394/233466390-67f5a789-8497-4cfe-830a-5b13e35074c9.png)

because it might offer some confusion

Reply:
> should it looks like this? 
> or is the button wrong?

Personally I like the buttons, they draw attention.

For the slogan, perhaps remove "Cloud native" from the front? So it's just "application lifecycle orchestration" then a user can pick cloud native or not?

Not sure if it is "Cloud-native" or "Cloud native". Most recent signage I've seen has it as "cloud native". Same for "life-cycle". I think it may be "lifecycle"? [Merriam Webster has it as "life cycle"](https://www.merriam-webster.com/dictionary/life%20cycle) so I'm leaning towards the separate versions